### PR TITLE
Render Skeleton When Loading Profile Viewer

### PR DESCRIPTION
**Pull request by hreineck**
Modified the way the profile viewer loads to show only the skeleton
card when the page is loading in; this fixes the issues with the
skeleton not rendering at all, as well as making it a bit more clear
that the page is still loading content.

### CODE REVIEW

**Comment by hreineck** [comment]
In the[previous commit](https://github.com/EasyDynamics/oscal-react-library/pull/114/commits/39b895773c3cd165b759756fbcd7ff1063215482) that initially sets up the skeleton,
there is an array of 3 skeleton cards rendered. I felt the single card looked a bit better, but if rendering multiple cards is desired that can be implemented instead.

File: src/components/OSCALProfile.js [hunk 1]
```diff
@@ -100,18 +77,31 @@ export default function OSCALProfile(props) {
...
+        <CardContent key="skeleton-card">
+          <span
+            style={{ marginTop: 5, display: "flex", gap: "1em" }}
+            key="controls load 0"
+          >
+            <Skeleton variant="text" width="25em" height="3em" />
+            <Skeleton variant="circle" width="3em" height="3em" />
+          </span>
+          <Skeleton variant="text" width="10em" height="2.5em" />
+          <Skeleton variant="rect" width="100%" height={115} />
```
Comment (rgauss) [inline src/components/OSCALProfile.js:101]: Can we see what it looks like with 2 cards that only contain this 100% width `rect`?

**Review by rgauss — CHANGES_REQUESTED** [review]

**Review by rgauss — CHANGES_REQUESTED** [review]
Sorry, looked better before my terrible suggestion :worried:.

**Review by rgauss — APPROVED** [review]
Looks good, provided that the distribution (commands the pipeline execute) displays things properly.

**Review by kkennedy26 — APPROVED** [review]
Comparing the two different styles, it definitely looks better this way having the skeleton for loading imported controls instead of the whole page. Also, it isn't part of this task, but there is a console error for a key prop when running tests. I don't know if there is an issue for it already.

**Comment by tuckerzp** [comment]
> Also, it isn't part of this task, but there is a console error for a key prop when running tests.

Out of curiosity, was console error there before this branch was created? Just don't want to unintentionally add errors to fix later.

**Comment by kkennedy26** [comment]
> Out of curiosity, was console error there before this branch was created? Just don't want to unintentionally add errors to fix later.

It must have been there before this branch, but not exactly sure where or when this error occurred.

**Review by mikeisen1 — APPROVED** [review]
This looks so much better with the skeleton rendering than without. Great work!

**Pull request merged**